SwiftLeeds/Extension/View+MeasureSize.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  View+MeasureSize+Extension.swift
+//  View+MeasureSize.swift
```

Comment:
I'm not totally sure it's necessary 
to go around and change all these.

Reply:
I didn't mean to add them in the initial colour example and didn't match the new extension I added, so standardised